.gitignore

```diff
@@ -11,6 +11,8 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+opcode.trace
```

Comment:
where do we use opcode.trace files?

Reply:
When wasmer crashes it prints out an opcode.trace file, I think it's a feature Camil made.